src/classes/services/ContactService.cls

```diff
@@ -27,18 +26,36 @@ public with sharing class ContactService {
 
     public static void updateTeamMember(Map<Id,Contact> oldContactById, Map<Id,Contact> newContactById){
```

Comment:
MINOR
Don`t forget about space before the method body ;)
Keep an eye on code style.